sqlglot/dialects/singlestore.py

```diff
@@ -136,6 +136,7 @@ class Parser(MySQL.Parser):
                 this=seq_get(args, 0),
                 format=MySQL.format_time(exp.Literal.string("%W")),
             ),
+            "APPROX_COUNT_DISTINCT": lambda args, dialect: exp.Hll(this=seq_get(args, 0)),
```

Comment:
Why is this only setting `this`? SingleStore supports multiple arguments, so I think we need `from_arg_list`:

```
APPROX_COUNT_DISTINCT ( expr [, expr ...] )
```